src/restaurant.js

```diff
@@ -71,6 +71,21 @@
 
 // PASSO 4: Adicione ao objeto retornado por `createMenu()` uma chave `pay` com uma função que varre todo os itens de `objetoRetornado.consumption`, soma o preço de todos checando-os no menu e retorna o valor somado acrescido de 10%. DICA: para isso, você precisará varrer tanto o objeto da chave `food` quanto o objeto da chave `drink`.
 
-const createMenu = () => {};
+const createMenu = (obj) => {
+  const objeto = {
+    fetchMenu: obj,
```

Comment:
```suggestion
    fetchMenu: () => obj,
```

O fetchMenu tem que retornar uma function